user_docs/en/changes.md

```diff
@@ -81,6 +81,8 @@ Specifically, MathML inside of span and other elements that have the attribute `
 In any document, if the cursor is on the last line, it will be moved to the end when using this command.
 (#17251, #17430, @nvdaes)
 * In web browsers, changes to text selection no longer sometimes fail to be reported in editable text controls. (#17501, @jcsteh)
+* When the Standard HID Braille Display driver is explicitly set as braille display driver and the braille display list is fetched again, NVDA reports the HID driver as the selected driver rather than no driver at all. (#17522, @LeonarddeR)
```

Comment:
I'm having difficulty parsing this - could you please try rewriting it for clarity?

Reply:
Is this better?
```suggestion
* When the Standard HID Braille Display driver is explicitly selected as the braille display driver, and the braille display list is opened, NVDA correctly identifies the HID driver as the selected driver instead of showing no driver selected. (#17522, @LeonarddeR)
```

Reply:
Yep, thanks @LeonarddeR